### PR TITLE
base twitter implementation

### PR DESCRIPTION
**Pull request by jackyzha0**

### CODE REVIEW

File: browser/src/index.css [hunk 1]
```diff
@@ -75,13 +75,7 @@ main {
...
   height: 100vh;
   width: 100vw;
-  background: radial-gradient(
-    at 50% 100%,
-    #ffd6db 0%,
-    #ffdade 40%,
-    #996296 80%,
-    #070710 100%
-  );
+  background: linear-gradient(to right, #fbc7d4, #9796f0);
```
Comment (spencerc99) [inline browser/src/index.css:78]: fyi @alejandrogarciasalas

File: browser/src/App.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ function App() {
         <div className="fadeOutOnScroll">
           <Hero />
         </div>
-        <div style={{ backdropFilter: `blur(3px)` }}>
+        <div>
```
Comment (spencerc99) [inline browser/src/App.tsx:59]: is this removing the glassmorphism?

File: server/api-handlers/twitter-verify.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        const handle = req.params.handle;
+        const { signature, walletId } = req.body;
+
+        const existingUser = await prisma.user.findFirst({
+            where: { id: walletId },
+        });
+
+        if (existingUser.twitterVerified) {
+            console.log(`already verified user: @${existingUser.twitterUsername}`);
+            res.json({ status: "ok", msg: "already verified"});
```
Comment (spencerc99) [inline server/api-handlers/twitter-verify.ts:29]: nit: 

```suggestion
            res.status(200).json({msg: "already verified"})
```

File: server/index.ts [hunk 4]
```diff
@@ -32,6 +37,10 @@ contributionsRouter.get("/:id", getContribution(services));
 contributionsRouter.post("/", addContribution(services));
 app.use("/contributions", contributionsRouter);
 
+const twitterRouter = express.Router();
+twitterRouter.post("/verify", verify(services));
```
Comment (spencerc99) [inline server/index.ts:41]: how does this work? does the user click a button on our website to verify after they've posted?

File: browser/src/App.tsx (hunk 2 above)
Comment (jackyzha0) [reply browser/src/App.tsx:59]: just for the essay section!

Comment (spencerc99) [reply browser/src/App.tsx:59]: ah gotcha i think we'll have problems with the blobs then but @alejandrogarciasalas if you want to look into something more performant that'd be great!

**Review by spencerc99 — APPROVED** [review]

File: server/index.ts (hunk 4 above)
Comment (jackyzha0) [reply server/index.ts:41]: Ideally this is part of the flow (similar to how Interdependence worked) before they officially post

File: browser/src/App.tsx (hunk 2 above)
Comment (alejandrogarciasalas) [reply browser/src/App.tsx:59]: yeah! I have a maxed out m1 mbp so things feel ok perf wise, will do some testing with throttling 👍

**Review by alejandrogarciasalas — APPROVED** [review]

**Pull request merged**